app/Models/Count.php

```diff
@@ -36,4 +38,11 @@ public static function totalUsers()
     {
         return static::find('usercount')->count ?? 0;
     }
+
+    public static function cachedTotalUsers()
+    {
+        return Cache::remember('count_totalUsers', 5, function () {
```

Comment:
does it actually save anything? Pretty sure the query itself is just selecting value of single row ಠ_ಠ